tests/models_core/test_magic_wand.py

```diff
@@ -25,6 +26,8 @@
 ]
 
 
+@pytest.mark.skipif(torch.cuda.get_device_capability() < (8, 0),
```

Comment:
Is this failing just because it runs OOM?

Reply:
yep